src/NuGet.Jobs.Db2AzureSearch/README.md

```diff
@@ -1 +1,122 @@
-TODO: https://github.com/NuGet/NuGetGallery/issues/8005
+## Overview
+
+**Subsystem: Search 🔎**
+
+This tool creates the resources needed to run the [NuGet search service](../NuGet.Services.SearchService). These resources can be updated using the [Catalog2AzureSearch](../NuGet.Jobs.Catalog2AzureSearch) and [Auxiliary2AzureSearch](../NuGet.Jobs.Auxiliary2AzureSearch) jobs.
+
+Specifically, this tool creates:
+
+* The [Azure Search indexes](../../docs/Azure-Search-indexes.md)
+* The [search auxiliary files](../../docs/Search-auxiliary-files.md)
+* The [search version list resource](../../docs/Search-version-list-resource.md)
+
+## Running the job
+
+You can run this job using:
+
+```ps1
+NuGet.Jobs.Db2AzureSearch.exe -Configuration path\to\your\settings.json
+```
+
+### Using DEV resources
+
+The easiest way to run the tool if you are on the nuget.org team is to use the DEV environment resources:
+
+1. Install the certificate used to authenticate as our client AAD app registration into your `CurrentUser` certificate store.
+1. Clone our internal [`NuGetDeployment`](https://nuget.visualstudio.com/DefaultCollection/NuGetMicrosoft/_git/NuGetDeploymentp) repository.
+1. Update your cloned copy of the [DEV Db2AzureSearch appsettings.json](https://nuget.visualstudio.com/DefaultCollection/NuGetMicrosoft/_git/NuGetDeployment?path=%2Fsrc%2FJobs%2FNuGet.Jobs.Cloud%2FJobs%2FDb2AzureSearch%2FDEV%2Fnorthcentralus%2Fappsettings.json) file to authenticate using the certificate you installed:
+```json
+{
+    ...
+    "KeyVault_VaultName": "PLACEHOLDER",
+    "KeyVault_ClientId": "PLACEHOLDER",
+    "KeyVault_CertificateThumbprint": "PLACEHOLDER",
+    "KeyVault_ValidateCertificate": true,
+    "KeyVault_StoreName": "My",
+    "KeyVault_StoreLocation": "CurrentUser"
+    ...
+}
+```
+
+1. Update the `-Configuration` CLI option to point to the DEV Azure Search settings: `NuGetDeployment/src/Jobs/NuGet.Jobs.Cloud/Jobs/Db2AzureSearch/DEV/northcentralus/appsettings.json`
+
+### Using personal Azure resources
+
+As an alternative to using nuget.org's DEV resources, you can also run this tool using your personal Azure resources.
+
+#### Prerequisites
+
+- **Gallery DB**. This can be initialized locally using the [NuGetGallery](https://github.com/NuGet/NuGetGallery/blob/master/README.md).
+- **Azure Search**. You can create your own Azure Search resource using the [Azure Portal](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal).
+- **Azure Blob Storage**. You can create your own Azure Blob Storage using the [Azure Portal](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create). You will need to create a container named `ng-search-data` and upload the following files:
+    1. [`downloads.v1.json`](https://nugetprod0.blob.core.windows.net/ng-search-data/downloads.v1.json)
```

Comment:
We should not document the explicit URLs but rather how to discover them from nugat deployment repo.

Reply:
Fixed. I also added "empty" content descriptions for that developers that aren't on the nuget.org team